bin/mkreleaselog

```diff
@@ -50,7 +85,7 @@ statlog() {
     fi
 
     local stack=()
-    git -C "$rpath" -c mailmap.file="$mailmap_file" log --use-mailmap --shortstat --no-merges --pretty="tformat:%H%x09%aN%x09%aE" "$start..$end" -- . ':^*\.pb\.go' ':^*ipldsch*\.go' ':^cbor_gen\.go\' ':^go\.mod' ':^go\.sum' | while read -r line; do
+    git -C "$rpath" -c mailmap.file="$mailmap_file" log --use-mailmap --shortstat --no-merges --pretty="tformat:%H%x09%aN%x09%aE" "$start..$end" -- . "${IGNORE_FILES_PATHSPEC[@]}" | while read -r line; do
```

Comment:
I don't think `IGNORE_FILES_PATHSPEC` usage is quite right. What we want is to ignore all files named go.mod, go.sum, etc. not just the ones in the root directory.

If you try this out by diffing v0.9.0 against v0.9.1 (by manually setting the start and end in the `recursive_release_log()` function) you'll see Marten having a huge diff that's due to updating go.sum and go.mod files in the go-libp2p examples repo. If you add a `.*` in front of go.mod and go.sum it goes away.

Reply:
My regex isn't great, but maybe something like `^(?:.*\/|)go\.mod$`

Reply:
That's strange... these are git "pathspecs" (not regexps) which should match at any point _unless_ they start with `/` (or `:/`). I'll dig into this.

Reply:
Git pathspecs aren't quite regex though. Not sure if there's a better way to write both `go\.mod` and `**/go\.mod` together

Reply:
Hm. You're right. The "starts with /" rule only applies when running git commands in subdirectories (i.e., matches files in the root instead of the subdirectory).

I've fixed this to match files anywhere.